doc/release_notes.rst

```diff
@@ -11,6 +11,8 @@ Release Notes
 Upcoming Release
 ================
 
+* Switch to new ``atlite`` version ``v0.2.1``. The version upgrade comes along with significant speed up for the rule ``build_renewable_profiles.py`` (~factor 2). A lot of the code which calculated the landuse availability is now outsourced and does not rely on ``glaes``, ``geokit`` anymore. This facilitates the environment building and version compatibility of ``gdal``, ``libgdal`` with other packages.     
```

Comment:
Switch to new major release, `>=v0.2.1`.

Reply:
done